src/bin/cargo/cli.rs

```diff
@@ -256,13 +256,19 @@ impl GlobalArgs {
 }
 
 fn cli() -> App {
+    let is_rustup = std::env::var_os("RUSTUP_HOME").is_some();
+    let usage = match is_rustup {
```

Comment:
Why a `match` not an `if`?

Reply:
Oh, no good reason. I think I wanted to save 1 line of code or something. I went ahead and switched it to a `if` since that is probably more natural.